### DISCUSSION TITLE
Integração com JPA Hibernate

### DISCUSSION THREAD

**Discussion by gabrielgeraldo** [wmixvideo/nfe#730] [Ideias]
Boa tarde pessoal!

Existe alguma forma de integrar o projeto com Hibernate? 
Por exemplo: Eu consigo mapear a classe NFNotaInfoItemImposto com seus relacionamentos?

A ideia é criar no banco de dados todos os campos da NFNotaInfoItemImposto com seus relacionamentos.

**Comment by fincatto** [comment]
Essa ideia pode quebrar o seu projeto. 
Conforme a lib vai evoluindo, novos campos são adicionados, removidos e até renomeados. 
O ideal, no meu ponto de vista, é que você faça esse mapeamento manualmente, sabendo o que você precisa.
